lib/helpers.js

```diff
@@ -4,7 +4,10 @@
  * https://github.com/bcoin-org/bpanel-utils
  */
 
-'use strict';
+const bio = require('bufio');
+const bcrypto = require('bcrypto');
+
+('use strict');
```

Comment:
I've never seen this syntax before, any reason its in parens?

Reply:
Oh weird. No idea. I wonder if prettier added that.

Reply:
Just tested. that's what looks like is happening